examples/website/data-filter/app.js

```diff
@@ -4,6 +4,9 @@ import {StaticMap} from 'react-map-gl';
 import DeckGL from '@deck.gl/react';
 import {ScatterplotLayer} from '@deck.gl/layers';
 import {DataFilterExtension} from '@deck.gl/extensions';
+import {Client as Styletron} from 'styletron-engine-atomic';
```

Comment:
This adds 10-20 lines of styletron-related code which seems like a lot of non-deck.gl-related "clutter" to add to example apps, that in my opinion should be as scaled-down as possible. 

Maybe this could go into the `wrapComponent` gatsby callback?

Reply:
I should probably comment this change more.  Firstly some necessary context: All of the apps in examples/website/* are built to be able to run standalone (by copying out the directory contents), as well as within the wrapping structure of the website.  This change refactors the placement of these specific instances of the BaseUI and Styletron providers so that they are **only** used in the standalone case.

More fundamentally, this change moves the BaseUI and Styletron providers out of the range-input.js component and up to the root of the standalone application, which is the recommended setup as described in the BaseUI docs https://baseweb.design/getting-started/setup/.  Prior to this, these root-level providers were implemented down inside of the range-input.js component.  Besides going against the recommend setup, this was causing issues when this example app was plugged into the new ocular-gatsby website, due to the fact that the application root of the ocular-gatsby site already had BaseUI and Styletron providers and duplicative providers was causing errors.  I've moved the clutter up from the actual UI component to the root of the application, but I think its necessary and correct in this case.

Reply:
> this change moves the BaseUI and Styletron providers out of the range-input.js component and up to the root of the standalone application

OK. Sounds like overall an improvement then. I am no expert on the styling libraries or the way they have been used in the deck.gl website.

I think what triggered my comment is that I saw only one example changed here. I assume the other examples do not have this code, and that it was just that one example that was using styletron providers inside a component so you moved them up, which is fine in that case (perhaps as a first step).